gc/stats/ConcurrentPhaseStatsBase.hpp

```diff
@@ -53,7 +53,8 @@ class MM_ConcurrentPhaseStatsBase : public MM_Base
 		terminationRequest_External
 	};
 	TerminationRequestType _terminationRequestType; /**< Reason for concurrent task termination, asynchronous external event or itself GC (work or survivor space exhausted etc). */
-
+	uint64_t _incrementStartTime;
```

Comment:
`_incrementStart/EndTime` is specific to scanvenger, these shouldn't be in this base class. Instead You can reuse generic members from `MM_ConcurrentMarkPhaseStats`:
```
	uint64_t _startTime;
	uint64_t _endTime;
```
These can be moved to base class and be used for scavenger